### ISSUE TITLE
Mask Customization Bug

### ISSUE THREAD

**Issue by Bwamo** [HugoZink/IreNFist#12]
Game crashes with this error after attempting to select a color during mask customization.
```
Application has crashed: C++ exception`
mods/IRE AND FIST REBORN/lua/blackmarketgui.lua:1253: attempt to index local 'part_info' (a nil value)

SCRIPT STACK

set_selected_tab() lib/managers/menu/blackmarketgui.lua:6984
mouse_pressed() lib/managers/menu/blackmarketgui.lua:6648
mouse_pressed() lib/managers/menu/menucomponentmanager.lua:1463
mouse_pressed() @mods/base/lua/MenuComponentManager.lua:24
mouse_pressed() lib/managers/menu/menurenderer.lua:372
mouse_press() lib/managers/menu/menuinput.lua:662
lib/managers/mousepointermanager.lua:345
```

**Comment by HugoZink** [comment]
It seems that the new PAYDAY update has radically changed the way mask color customization works. I'll get right on it.

**Comment by HugoZink** [comment]
Fixed in 2f12413cef429bf6f4c0804f1aaee92b79c4d6c8.

BlackMarketGUI is probably the biggest source of issues because the affected function is literally thousands of lines long, any mod or any update that makes a slight change is bound to become incompatible with InF. I might have to give this function another pass later, there might be a better way to do this. All the function really changes is show weapon and weaponmod descriptions.
